.github/actions/download_cppsdk/powershell/action.yml

```diff
@@ -1,22 +1,19 @@
 name: 'CppSDKReleaseAssetDownloader'
 description: 'Download specific C++ SDK release asset.'
 inputs:
-  url:
-    description: 'C++ SDK release URL.'
-    required: true
   tag:
     description: 'Release tag of C++ SDK release.'
     required: true
-  asset_name:
-    description: 'Define name of asset'
+  asset_suffix:
+    description: 'The suffix of the C++ SDK asset defining the platform.'
     required: true
 runs:
   using: 'composite'
   steps:
     - name: Download C++ Windows
       run: |
-        $Env:INPUT_URL = "${{ inputs.url }}"
+        $Env:INPUT_URL = "https://github.com/DolbyIO/comms-sdk-cpp/releases/download"
```

Comment:
i think better idea is to leave it on top configuration in CPP_SDK_URL variable in build.yml it will be easier to fix if something will change in ~long~ time